datalad/distribution/dataset.py

```diff
@@ -36,9 +36,10 @@
     InvalidGitRepositoryError,
     NoSuchPathError
 )
-from datalad.support.repo import Flyweight
+from datalad.support.repo import PathBasedFlyweight
 from datalad.support.network import RI
 from datalad.support.exceptions import InvalidAnnexRepositoryError
+from datalad.support import path as op
```

Comment:
This is introducing a duplicate import of `op`.

Reply:
rright -- removing `import os.path as op` above. pushed 37e9a9e05599b271cc5743f3cb0155babcf9c197